src/docs/asciidoc/user-guide/assertj-guava-assertions-guide.adoc

```diff
@@ -0,0 +1,104 @@
+|===
+|Assertion |Description
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#contains(org.assertj.core.data.MapEntry...)[`contains`]
+|Verifies that the actual `Multimap` contains the given entries.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#containsAllEntriesOf(com.google.common.collect.Multimap)[`containsAllEntriesOf`]
+|Verifies that the actual `Multimap` contains all entries of the given one (it might contain more entries).
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#containsKeys(K...)[`containsKeys`]
+|Verifies that the actual `Multimap` contains the given keys.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#containsValues(V...)[`containsValues`]
+|Verifies that the actual `Multimap` contains the given values for any key.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#hasSameEntriesAs(com.google.common.collect.Multimap)[`hasSameEntriesAs`]
+|Verifies that the actual `Multimap` has the same entries as the given one.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#hasSize(int)[`hasSize`]
+|Verifies that the number of values in the actual `Multimap` is equal to the given one.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#isEmpty()[`isEmpty`]
+|Verifies that the actual `Multimap` is empty.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultimapAssert.html#isNotEmpty()[`isNotEmpty`]
+|Verifies that the actual `Multimap` is not empty.
+|===
+
+==== Multiset
+
+On top of `Iterable` link:#assertj-core-iterable-assertions[assertions], the following are also available.
+
+[%autowidth]
+|===
+|Assertion |Description
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultisetAssert.html#contains(int,T)[`contains`]
+|Verifies the actual `Multiset` contains the given value *exactly* the given number of times.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultisetAssert.html#containsAtLeast(int,T)[`containsAtLeast`]
+|Verifies the actual `Multiset` contains the given value *at least* the given number of times.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/MultisetAssert.html#containsAtMost(int,T)[`containsAtMost`]
+|Verifies the actual `Multiset` contains the given value *at most* the given number of times.
+|===
+
+==== Optional
+
+[%autowidth]
+|===
+|Assertion |Description
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/OptionalAssert.html#contains(java.lang.Object)[`contains`]
+|Verifies that the actual `Optional` contains the given value.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/OptionalAssert.html#isAbsent()[`isAbsent`]
+|Verifies that the actual `Optional` contained instance is absent/null.
+
+|{assertj-guava-javadoc-root}org/assertj/guava/api/OptionalAssert.html#isPresent()[`isPresent`]
+|Verifies that the actual `Optional` contains a (non-null) instance.
+|===
+
+[%autowidth]
+|===
+|--FIXME-- |Description
```

Comment:
@joel-costigliola I was thinking about separating normal assertion methods from "extracting" / "navigational" methods. Any good naming for them?

Reply:
Maybe "advanced usage" or "asserting on the contained value" ? 
the first is generic and not too descriptive, I kinda prefer the second but no strong opinion here

Reply:
I'm looking for something that could also cover the various `extracting`, `first`, `last`, `asInstanceOf`, etc.

Reply:
I'd put them together in a single table for the time being, going forward with the rest of the docs maybe a good name will pop up.